docs/reference/migration/migrate_2_0/settings.asciidoc

```diff
@@ -1,5 +1,24 @@
 === Setting changes
 
+==== Command line flags
+
+Command line flags using notation `-Des.*` must be now specified as the first arguments.
```

Comment:
Its actually anything that starts with a single dash I believe. Which is mostly `-D`.

The rest of the change is great.